capstone/scripts/ingest_tt_data.py

```diff
@@ -2,19 +2,19 @@
 
 from tqdm import tqdm
 
-from cap.models import VolumeMetadata, TrackingToolUser, Reporter, ProcessStep, TrackingToolLog, BookRequest
+from capdb.models import VolumeMetadata, TrackingToolUser, Reporter, ProcessStep, TrackingToolLog, BookRequest
 from tracking_tool.models import BookRequests, Eventloggers, Hollis, Pstep, Reporters, Users, Volumes
 
 
 user_field_map = {'id': 'id', 'privlevel': 'privilege_level', 'email': 'email', 'created_at': 'created_at', 'updated_at': 'updated_at' }
 book_request_field_map = {'id': 'id', 'updated_by': 'updated_by', 'created_at': 'created_at', 'updated_at': 'updated_at', 'recipients': 'recipients', 'from_field': 'from_field', 'mail_body': 'mail_body', 'note': 'note', 'send_date': 'send_date', 'label': 'label', 'sent_at': 'sent_at', 'subject': 'subject', 'delivery_date': 'delivery_date'} 
 pstep_field_map = {'step_id': 'step', 'name': 'label', 'prereq': 'prerequisites', 'desc': 'description', 'created_at': 'created_at', 'updated_at': 'updated_at'}
-reporter_field_map= {'id': 'id', 'state': 'jurisdiction', 'reporter': 'full_name', 'short': 'short_name', 'start_date': 'start_year', 'end_date': 'end_year', 'volumes': 'volume_count', 'created_at': 'created_at', 'updated_at': 'updated_at', 'notes': 'notes'} 
```

Comment:
removed from mapping because reporter now has ManyToMany Jurisdiction field, which needs to be set like this: reporter_instance.jurisdictions.add(new_jurisdiction)